From 87bf44fe091bdf446aa00e052dd10418a379a1a6 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Tue, 31 May 2016 16:12:38 -0400
Subject: [PATCH 12/12] systemd: disable GVFS support

accountsservice doesn't need support for GVFS modules,
and those modules lead to /root/.cache getting created
at startup.

accountsservice runs as root, but shouldn't be touching /root.

This commit disables GVFS for accountsservice

https://bugs.freedesktop.org/show_bug.cgi?id=96316
---
 data/accounts-daemon.service.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/accounts-daemon.service.in b/data/accounts-daemon.service.in
index feedf3e..2f5fe33 100644
--- a/data/accounts-daemon.service.in
+++ b/data/accounts-daemon.service.in
@@ -12,6 +12,9 @@ Type=dbus
 BusName=org.freedesktop.Accounts
 ExecStart=@libexecdir@/accounts-daemon
 StandardOutput=syslog
+Environment=GVFS_DISABLE_FUSE=1
+Environment=GIO_USE_VFS=local
+Environment=GVFS_REMOTE_VOLUME_MONITOR_IGNORE=1
 
 [Install]
 # We pull this in by graphical.target instead of waiting for the bus
-- 
1.9.5

